purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -406,6 +423,15 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
      * @param [skuDetails] The skuDetails of the product you wish to purchase
      * @param [listener] The listener that will be called when purchase completes.
      */
+    @Deprecated(
```

Comment:
is this a case where we might want to add multiple combinations of "replacement order" like [here](https://github.com/RevenueCat/purchases-android/pull/395#discussion_r762386632)? or is it not the same when they're both in the parameter list?

Reply:
Maybe! I need to try again these deprecations...